x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/_suggestion_panel.scss

```diff
@@ -48,6 +49,10 @@
   height: $lnsSuggestionHeight - $euiSizeL;
 }
 
+.lnsSuggestionPanel__chartWrapper > * {
```

Comment:
Instead of this universal selector, you just need to move the `.lnsXyExpression__chart` from the `.echChart` to the new div.